webbpsf/opds.py

```diff
@@ -2164,3 +2273,119 @@ def test2_OPDbender(filename='OPD_RevV_nircam_132.fits'):
     plt.subplot(133)
     diff = (perturbed - orig)
     diff.draw(title='Difference ({0:.1f} nm rms)'.format(diff.rms()), **plot_kwargs)
+
+
+#-------------------------------------------------------------------------------
+# Thermal
+from copy import deepcopy
+from . import optics
+from scipy import io
+
+
+class OteThermalModel(object):
+    """
+    Given a time in units of seconds, minutes, hours, or days.
```

Comment:
Revise this doc string to start with a one-sentence summary of what this class does.